src/components/AddMovie/AddMovie.css

```diff
@@ -0,0 +1,16 @@
+.button--transparent {
```

Comment:
If you are using BEM I would recommend to you to setup and use SASS

Reply:
Thank you! For now I will stay with regular css approach however I will consider it with the next project :)